### PR TITLE
[2] Execution price on Limit orders confirmation screen

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

image

### CODE REVIEW

File: src/cow-react/modules/limitOrders/updaters/InitialPriceUpdater/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { useLayoutEffect, useState } from 'react'
```
Comment (shoom3301) [inline src/cow-react/modules/limitOrders/updaters/InitialPriceUpdater/index.tsx:1]: The file just was renamed from `MarketPriceUpdater`

File: src/cow-react/modules/limitOrders/updaters/MarketPriceUpdater/index.tsx [hunk 2]
```diff
@@ -1,49 +1,7 @@
-import { useLayoutEffect, useState } from 'react'
-import { useUpdateAtom } from 'jotai/utils'
-import { LimitRateState, updateLimitRateAtom } from '@cow/modules/limitOrders/state/limitRateAtom'
-import { useGetInitialPrice } from '@cow/modules/limitOrders/hooks/useGetInitialPrice'
-import { useLimitOrdersTradeState } from '../../hooks/useLimitOrdersTradeState'
-import usePrevious from '@src/hooks/usePrevious'
-import { useUpdateActiveRate } from '@cow/modules/limitOrders/hooks/useUpdateActiveRate'
-import { Writeable } from '@cow/types'
+import { useFetchMarketPrice } from './hooks/useFetchMarketPrice'
```
Comment (shoom3301) [inline src/cow-react/modules/limitOrders/updaters/MarketPriceUpdater/index.tsx:1]: The file just was renamed from `QuoteUpdater`

**Comment by elena-zh** [comment]
Hey @shoom3301 , great job! 
Some questions and issues: 
1. Arrow in the light mode is almost invisible
image
2. When the modal is opened, the est.execution price is not fixed (it is still dynamically changed). Is this OK?
3. When  a price is reverted on the Limit order form, I believe it should be reverted on the order confirm modal. WDYT?
reverted
4. And it would be great to show fee in the same currency (reported in https://github.com/cowprotocol/cowswap/pull/2024#issuecomment-1434725390, case 2)
image
5. Est.execution price field should be left-aligned on the tooltip (see the image above).

Thank you!

**Comment by shoom3301** [comment]
@elena-zh thanks for your comments!
2 - it's ok, because the market is moving even when the modal is open
3 - the price inversion depends on the control of the modal (the icon to the right of the "Limit price")

**Review by elena-zh — APPROVED** [review]
Approved then :)

**Pull request merged**